src/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -25,11 +30,14 @@ import DATA_IMPORT_BATCH_GIFT_INFO from '@salesforce/schema/DataImportBatch__c.G
 import DATA_IMPORT_BATCH_DEFAULTS_INFO from '@salesforce/schema/DataImportBatch__c.Batch_Defaults__c';
 import DATA_IMPORT_MATCHING_BEHAVIOR_INFO from '@salesforce/schema/DataImportBatch__c.Donation_Matching_Behavior__c';
 import DATA_IMPORT_BATCH_TABLE_COLUMNS_FIELD from '@salesforce/schema/DataImportBatch__c.Batch_Table_Columns__c';
+import DATA_IMPORT_BATCH_DONATION_MATCHING_RULE from '@salesforce/schema/DataImportBatch__c.Donation_Matching_Rule__c';
 
 const NAME = 'name';
 const ID = 'id';
 const MAX_STEPS = 2;
 const CANCEL = 'cancel';
+const COMMA_SEPARATOR = ';';
+const NAMESPACE_PREFIX = 'npsp__';
```

Comment:
@voduyemi If there's any way we can avoid hard-coding the namespace here that would be great ... I realize this update is following my feedback on the previous iteration of this work, but I would love see if we can avoid the hard-coded reference here. @psadie-sfdo is this a case that we've addressed in the past, any advice here?

Reply:
@ptewson-sfdo I think the problem here is in a non-namespaced org, I'm not sure we have a way of retrieving the namespace prefix programatically and we need this namespace prefix specifically in a non-namespaced org. I'm open to suggestions on how to it can be achieved if anyone knows how.